Sources/HummingbirdCore/Server/ChildChannel.swift

```diff
@@ -16,15 +16,15 @@ import Logging
 import NIOCore
 
 /// HTTPServer child channel setup protocol
-public protocol HBChannelSetup: Sendable {
+public protocol HBChildChannel: Sendable {
```

Comment:
What about `HBChannelBootstrap`?

Reply:
Yeah I'm not sure about that. This only deals with the setup and handling of messages on a child channel. HBServer is the bootstrap.